tools/twix/src/panels/map/layers/behavior_simulator.rs

```diff
@@ -39,6 +49,34 @@ impl Layer for BehaviorSimulator {
                 color: Color32::BLACK,
             };
             painter.pose(robot_to_field, 0.15, 0.25, pose_color, pose_stroke);
+
+            let Ok(motion_command): Result<MotionCommand> = self.motion_command.0[player_number].parse_latest() else {
+                continue
+            };
+
+            if let MotionCommand::Walk { path, .. } = motion_command {
+                for segment in path {
+                    match segment {
+                        PathSegment::LineSegment(line_segment) => painter.line_segment(
+                            robot_to_field * line_segment.0,
+                            robot_to_field * line_segment.1,
+                            Stroke {
+                                width: 0.025,
+                                color: Color32::from_rgba_unmultiplied(0, 0, 255, 63),
+                            },
+                        ),
+                        PathSegment::Arc(arc, orientation) => painter.arc(
+                            arc,
+                            orientation,
+                            Stroke {
+                                width: 0.025,
+                                color: Color32::from_rgba_unmultiplied(0xAD, 0xD8, 0xE6, 63),
```

Comment:
Why do you mix hexadecimal and decimal numbers for your colors? Please choose one and stick to it. I'd vote for decimals.

Also, from looking at the code, it is not obvious which color this is, consider defining a named constant and using that here instead.